ingenialink/ethercat/dictionary.py

```diff
@@ -51,6 +56,27 @@ def _monitoring_disturbance_registers(self) -> list[EthercatRegister]:
             ),
         ]
 
+    @cached_property
+    def _safety_modules(self) -> list[DictionarySafetyModule]:
+        return [
+            DictionarySafetyModule(
+                uses_sra=False,
+                module_ident=int("0x3800000", 16),
```

Comment:
I think we should create variables for the module ident values.

Reply:
They are not decontextualized